src/commands/guild/disable.command.ts

```diff
@@ -0,0 +1,123 @@
+import { Context } from 'detritus-client/lib/command';
+import { User, Reaction } from 'detritus-client/lib/structures';
+import { ReactionCollector } from '../../modules/collectors/reactionCollector';
+import { QueryType } from '../../modules/db';
+
+interface CommandArgs {
+  'disable command': string;
+}
+
+export const disableCommand = {
+  name: 'disable command',
+  metadata: {
+    description: 'Disables or enables commands.',
+    checks: ['userAdmin'],
+  },
+  run: async (ctx: Context, args: CommandArgs) => {
+    if (!args['disable command']) {
+      ctx.reply(
+        `Usage: ${ctx.prefix}disable command {command to disable} Doing ${ctx.prefix}disable command tags will disable tag usage.`
+      );
+      return;
+    }
+    if (args['disable command'].toLowerCase() === 'disable command') {
```

Comment:
Just a stylistic thing, I'd be preferred if you stored `args['disable command'].toLowerCase()` in a var and just referenced that instead all the places.